### PR TITLE
Devdocs bootstrap forms

### PR DESCRIPTION
**Pull request by roland04**

### CODE REVIEW

File: docs/devupdate.md [hunk 1]
```diff
@@ -40,6 +40,90 @@ will only update the second enrol instance.
 
 A new method `enrol_plugin:find_instance()` is added to the enrol plugin interface to allow plugins to find an existing instance of the same type in the course. If you want your enrolment method to be supported in CSV course upload, you need to implement this [method](./apis/plugintypes/enrol#enrol_pluginfind_instance-stdclass)
 
+## Bootstrap preparations for version 5
+
+<Since version="4.4" issueNumber="MDL-71979" />
+
+Some of the Bootstrap 4 classes will deprecated or dropped in its version 5. To prepare for this, some of the current Bootstrap 4 classes usages have been replaced with version 5 compatible classes. This will help us to upgrade to Bootstrap 5 in the future.
```
Comment (ilyatregubov) [inline docs/devupdate.md:47]: Tiny comment - 'will deprecated' should be 'will be deprecated'

**Review by ilyatregubov — CHANGES_REQUESTED** [review]
Tiny comment - 'will deprecated' should be 'will be deprecated'

File: docs/devupdate.md (hunk 1 above)
Comment (roland04) [reply docs/devupdate.md]: Thanks a lot Ilya! Fixed.

**Review by ilyatregubov — APPROVED** [review]

**Comment by andrewnicols** [comment]
Note: This issue has not landed and this PR should not have been merged as a result.

**Comment by ilyatregubov** [comment]
Thanks @andrewnicols - sorry about that. I have reverted the changes. @roland04 could you please create a new merge request?

**Comment by roland04** [comment]
Sorry @andrewnicols @ilyatregubov. It was my fault. It should have been a draft from the beginning :pray: 
Thanks @andrewnicols to mark as Draft the rest of the related issues

**Pull request merged**